tests/.eslintrc

```diff
@@ -19,6 +19,10 @@
       // Allow dev-dependencies in this directory.
       "devDependencies": true
     }],
+    "jest/expect-expect": ["warn", {
```

Comment:
Should probably be an error if the config below is accurate.

Reply:
Unfortunately this configuration doesn't fix all violations of `jest/expect-expect`.  You can see that [the `master` branch has 26 violations](https://app.circleci.com/pipelines/github/mozilla/addons-linter/86/workflows/a1a3c74c-6cdb-4619-8b18-191856073147/jobs/207/parallel-runs/0/steps/0-109), and [this feature branch still has 12 violations](https://app.circleci.com/pipelines/github/mozilla/addons-linter/53/workflows/c4f2a855-10fa-45be-9f53-ed3fa9f1a6b8/jobs/127/parallel-runs/0/steps/0-109).

Most of the remaining `jest/expect-expect` violations have `expect()` assertions that the lint rule doesn't see, like https://github.com/mozilla/addons-linter/blob/247c3b02308eee34c257f2d2cad6a8888d7fe159/tests/unit/parsers/test.manifestjson.js#L2730-L2732 and https://github.com/mozilla/addons-linter/blob/247c3b02308eee34c257f2d2cad6a8888d7fe159/tests/unit/schema/test.background.js#L9-L17  Those could be permitted by adding the helper function names to `assertFunctionNames` like
```json
"assertFunctionNames": ["expect", "sinon.assert.*", "testHomepageUrl", "assertHasMatchingError"]
````

And some of the violations are indeed assertion-less tests like https://github.com/mozilla/addons-linter/blob/247c3b02308eee34c257f2d2cad6a8888d7fe159/tests/unit/scanners/test.html.js#L105-L110

Basically I noticed that my new test was adding an ESLint warning (because it doesn't have `expect()`) and I wanted to avoid that.  But as a first-time contributor, I figure you probably don't want me making involved changes to your ESLint config 😃.  I'm happy to revert the ESLint change if it simplifies this PR.

Reply:
it's probably fine, thanks!